requirements.in

```diff
@@ -7,3 +7,4 @@ pyyaml
 rauth
 requests
 scikit-learn
+cachetools
```

Comment:
just to be absolutely sure that this does not make problems later on some server, please add a newline